shared/settings/notifications/index.js

```diff
@@ -91,6 +94,19 @@ const Notifications = (props: Props) =>
             unsubscribedFromAll={props.groups.app_push.unsubscribedFromAll}
           />}
 
+        {props.groups.security &&
+          props.groups.security.settings &&
+          <Group
+            allowEdit={props.allowEdit}
+            groupName="security"
+            onToggle={props.onToggle}
+            onToggleUnsubscribeAll={() => {}}
```

Comment:
i don't think its too verbose for the other side to do 
```
this.props.onToggleUnsubscribeAll && this.props.onToggleUnsubscribeAll()
```

and forcing all users of a component to fill in props they don't need isn't ideal. also this will lead to more render thrashing as this closure will always fail pure comparison tests